src/StringCompression.cpp

```diff
@@ -12,7 +12,6 @@
 
 
 
-
```

Comment:
Please don't erase empty lines between functions (style: there are to be 5 empty lines between function definitions). Make sure you run the `src/CheckBasicStyle.lua` script before committing.

Reply:
Ok, I will check the pre-commit hook.
I am not sure why it did not run but I will run the pre-commit hook manually next time before the next commit.
The deleted line will be re-added to allow for 5 lines between function definitions.

Reply:
You need to explicitly enable the pre-commit hook on your local machine (and install Lua, if you haven't already). This is a security measure in git, so that foreign repositories can't run arbitrary code on your computer by putting it into a hook.

If you need a Lua interpreter for Windows, my [LunaPaak](https://github.com/madmaxoft/LunaPaak) project could help you - it's a single executable that you download and run once, it will register itself as Lua interpreter, and comes with several libraries built in, so it's pretty good for scripting.